<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>672</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #1E1E1E;
}

QWidget {
    background-color: #252525;
    color: #E0E0E0;
}

QLabel {
    color: #E0E0E0;
}

QTableWidget {
    background-color: #252525;
    border: none;
    gridline-color: #333;
    alternate-background-color: #282828;
}

QHeaderView::section {
    background-color: #2D2D2D;
    color: #CCCCCC;
    padding: 6px;
    border: none;
    font-weight: bold;
}

QTableWidget::item:selected {
    background-color: #3A3A3A;
    color: #FFFFFF;
}

QLineEdit {
    background-color: #2D2D2D;
    color: #E0E0E0;
    border: 1px solid #333;
    border-radius: 6px;
    padding: 6px;
}

#kickButton {
    background-color: #FF5252;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 6px 16px;
    font-weight: bold;
}
#kickButton:hover {
    background-color: #FF3B3B;
}

#banButton {
    background-color: #FF9800;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 6px 16px;
    font-weight: bold;
}
#banButton:hover {
    background-color: #FF8A00;
}

#unbanButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 6px 16px;
    font-weight: bold;
}
#unbanButton:hover {
    background-color: #43A047;
}

QPushButton {
    background-color: #34495E;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 6px 16px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #2C3E50;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="kickButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Кик</string>
    </property>
   </widget>
   <widget class="QPushButton" name="banButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Бан</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>789</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Добро пожаловать в мониторинг серверной части! Вот всё вам необходимое, чтобы сжигать еретиков:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unbanButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Разбан</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="usersTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1081</width>
      <height>201</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Login</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nickname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ban</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="messagesTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>1081</width>
      <height>271</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
    </property>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sender</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Recipient</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Content</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
